ietf/doc/templatetags/ietf_filters.py

```diff
@@ -779,3 +779,10 @@ def is_valid_url(url):
     except ValidationError:
         return False
     return True
+
+@register.filter
+def get_hash(h, key):
```

Comment:
Suggest naming this `get_from_dict` or something that better matches the dict Python data type. (There's a similar filter `lookup` in `agenda_custom_tags.py` that could perhaps be moved and reused - this is a better place for it.)